disabled-packages/ghc/build.sh

```diff
@@ -3,10 +3,10 @@
 TERMUX_PKG_HOMEPAGE=https://www.haskell.org/ghc/
 TERMUX_PKG_DESCRIPTION="The Glasgow Haskell Compilation system"
 TERMUX_PKG_LICENSE="BSD 2-Clause, BSD 3-Clause, LGPL-2.1"
-TERMUX_PKG_VERSION=8.10.1
-TERMUX_PKG_REVISION=5
+TERMUX_PKG_VERSION=9.2.1
+TERMUX_PKG_REVISION=6
```

Comment:
You should remove `TERMUX_PKG_REVISION` as version has been updated
```suggestion
```

Reply:
Also move this form `disabled-packages/` to `packages/`

Reply:
Ah fair enough. Wasn't sure it was getting used.

Reply:
I don't know about the fix - need to try that out.